docs/source/vcs.rst

```diff
@@ -0,0 +1,73 @@
+===================================
+Verifiable Credential Service (VCS)
+===================================
+
+What is a Verifiable Credential (VC)?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use credentials everyday. A driver's license issued by the government certify
+that we are capable of operating a vehicle on the road. A Permanent Residence card
+shows the immigration status of an individual.
+
+A verifiable credential is then a document whose contents can be proven to be true. 
```

Comment:
May be we can add `cryptographically proven/verified` here - https://www.w3.org/TR/vc-data-model/#terminology

Reply:
added @rolsonquadras